applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -77,7 +83,17 @@ static struct bt_audio_stream *lc3_cap_config_cb(struct bt_conn *conn, struct bt
 	if (!stream->conn) {
 		LOG_INF("ASE Codec Config stream %p", (void *)stream);
 
-		print_codec(stream->codec);
+		print_codec(codec);
+
+		uint32_t octets_per_sdu = bt_codec_cfg_get_octets_per_frame(codec);
+
+		if (octets_per_sdu > LE_AUDIO_SDU_SIZE_OCTETS(CONFIG_LC3_BITRATE_MAX)) {
+			LOG_WRN("Too high bitrate");
```

Comment:
Should we using LOG_ERR instead of LOG_WRN? Since if the setting is out of support then we won't return stream.

Reply:
Probably we can have another ticket for that feature later.